jobs/mesoscale/stats/JEVS_MESOSCALE_STATS

```diff
@@ -38,23 +38,29 @@ export SENDDBN=${SENDDBN:-YES}
 export SENDECF=${SENDECF:-YES}
 
 export DATA=${DATA:-${DATAROOT:?}/${jobid:?}}
-rm -rf $DATA
 mkdir -p $DATA
 cd $DATA
 
-
+export machine=${machine:-WCOSS2}
+export MET_bin_exec=${MET_bin_exec:-bin}
```

Comment:
This line (45) is not needed as it is defined in `parm/metplus_config/machine.conf`